src/main/java/module-info.java

```diff
@@ -1,4 +1,7 @@
 module net.jonathangiles.tools.teenyhttpd {
     exports net.jonathangiles.tools.teenyhttpd;
     exports net.jonathangiles.tools.teenyhttpd.model;
+
+    requires java.logging;
+    requires java.desktop;//required for @Transient
```

Comment:
I really don't want the java.desktop module dependency - can we please get rid of it?

Reply:
Sure